bin/sdk-cli.js

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env node
+
+/** @format */
+
+const chalk = require( 'chalk' );
+const execSync = require( 'child_process' ).execSync;
+const path = require( 'path' );
+
+const buildBlockScript = path.resolve( __dirname, 'create-scripts/block.js' );
+const crossEnvShell = path.resolve( __dirname, '../node_modules/.bin/cross-env-shell' );
+let args = process.argv;
+
+// Remove `node` and SDK script
+args.splice(0, 2);
+
+// First argument should always be the task name
+const task = args.shift();
+
+if ( ! args.length || task !== 'build-block' ) {
+	console.log( chalk.red( 'usage: npx calypso-gutenberg-sdk build-block <block>' ) );
+	process.exit(1);
+}
+
+execSync( `${ crossEnvShell } SKIP_FLAG_IMAGES=true node ${ buildBlockScript } ${ args.join( ' ' ) }`, {
```

Comment:
Now, I'd prefer to just pass our npm script (`npm run build-block ...`) directly to `execSync()` since that would be much cleaner but I didn't see any obvious way to do so: `execSync()` runs relative to `npx`, e.g. to Jetpack's root folder.

This whole file will get re-written in a follow-up PR very soon, so I don't mind having this a bit hacky at this point.

The main purpose of this file right now is just to solidify this API and unblock further cli-explorations: `npx calypso-gutenberg-sdk build-block ...`

Reply:
Eventually, we'll probably just do something like this instead:

```js
import { build } from 'sdk/block.js';

build( args );
```